Sources/NetworkProtection/Diagnostics/NetworkProtectionError.swift

```diff
@@ -93,15 +97,19 @@ public enum NetworkProtectionError: LocalizedError, CustomNSError {
         case .failedToFetchLocationList: return 102
         case .failedToParseLocationListResponse: return 103
         case .failedToEncodeRegisterKeyRequest: return 104
-        case .failedToFetchRegisteredServers: return 105
-        case .failedToParseRegisteredServersResponse: return 106
-        case .failedToEncodeRedeemRequest: return 107
-        case .invalidInviteCode: return 108
-        case .failedToRedeemInviteCode: return 109
-        case .failedToRetrieveAuthToken: return 110
-        case .failedToParseRedeemResponse: return 111
-        case .invalidAuthToken: return 112
-        case .serverListInconsistency: return 113
+        case .noResponseFromRegisterEndpoint: return 105
+        case .unexpectedStatusFromRegisterEndpoint: return 106
```

Comment:
Added these two

Reply:
I notice that `failedToFetchRegisteredServers` has changed from code 105 to 107, is that intentional?

Reply:
As discussed via DM, I made this change as these errors are still settling and this info is still not really coming up.

But we should definitely avoid this in the future.